websites/W/Worldhideaway/presence.ts

```diff
@@ -2,151 +2,177 @@ const presence = new Presence({
+					},
+				];
+				presenceData.details = "Leyendo FAQ";
 				presenceData.smallImageKey = "reading";
-			} else if (page === "/contacto/") {
-				presenceData.details = "Viendo:";
-				presenceData.state = "Contacto";
-			} else if (expage.includes("/wp-login.php?action=lostpassword")) {
-				presenceData.details = "Viendo:";
-				presenceData.state = "Contraseña Olvidada";
-			} else if (expage === "https://worldhideaway.com/wp-login.php")
-				presenceData.details = "Conectar";
-			else {
-				switch (page) {
-					case "/register/": {
-						presenceData.details = "Registro";
-						break;
-					}
-					case "/logros/": {
-						presenceData.details = "Viendo:";
-						presenceData.state = "Logros";
-
-						break;
-					}
-					case "/events/": {
-						presenceData.details = "Viendo:";
-						presenceData.state = "Events";
-
-						break;
-					}
-					default:
-						if (page.includes("/event/")) {
-							title = document.querySelector(
-								"#tribe-events-content > div.bs-event-heading > div.tribe-event-schedule-long > div.bs-tribe-events-single-heading > h1"
-							);
-							presenceData.details = "Viendo:";
-							presenceData.state = title.textContent;
-						} else {
-							switch (page) {
-								case "/equipo/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Equipo";
-
-									break;
-								}
-								case "/texto/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Texto";
-
-									break;
-								}
-								case "/gestos/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Gestos";
-
-									break;
-								}
-								case "/stickers/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Pegatinas";
-
-									break;
-								}
-								case "/salas/": {
-									presenceData.details = "Viendo:";
-									presenceData.state = "Salas";
+				break;
+			}
+			case "contacto": {
+				presenceData.buttons = [
+					{
+						label: "Viendo Contacto",
+						url: href,
+					},
+				];
+				presenceData.details = "Viendo contacto";
+				break;
+			}
+			default: {
+				const check = document.querySelector(
+						"#main > div > div > section.elementor-section.elementor-top-section.elementor-element.elementor-element-7cf6a1d.elementor-section-boxed.elementor-section-height-default.elementor-section-height-default > div > div > div > div > div > div > div > h1"
+					),
+					active =
+						document.querySelector('[aria-current="page"]') ??
+						document.querySelector('[class="current_page_item"]');
 
-									break;
-								}
-								// No default
-							}
-						}
-				}
+				if (check) {
+					presenceData.buttons = [
+						{
+							label: "Leyendo",
+							url: href,
+						},
+					];
+					presenceData.details = `Leyendo ${check.textContent}`;
+					presenceData.smallImageKey = "reading";
+				} else if (active) {
+					presenceData.buttons = [
+						{
+							label: "Viendo",
```

Comment:
The label is the text that shows up on the button. Its purpose is to perform an action so you should be using the infinitive form instead of the gerund (Ver ...), try setting something more specific. What would you be looking at?

Reply:
A menu’s topic

Reply:
So the sentence should be: "Viewing Information"